tutorials/aws-firehose-s3/consumer.py

```diff
@@ -0,0 +1,163 @@
+import glassflow
+import boto3
+import sys
+import time
+import json
+from dotenv import dotenv_values
+
+config = dotenv_values(".env")
+
+pipeline_id = config.get("PIPELINE_ID")
+space_id = config.get("SPACE_ID")
```

Comment:
After internal changes of `pipeline_id`, we are currently unsure if we need `space_id` for SDK anymore.
@ashish-bagri is investigating this topic.

Reply:
@vkd Thanks for reporting this! 

This demo code will be not merged anyway. When we have a big announcement, we should update this change everywhere:) This example using spaceId is now everywhere.